tuf/utils/x509_test.go

```diff
@@ -239,6 +279,296 @@ func TestECDSAX509PublickeyID(t *testing.T) {
 	require.Equal(t, tufPrivKey.ID(), tufID)
 }
 
+func preserveEnv(name string) func() {
+	if env, has := os.LookupEnv(name); has {
+		os.Unsetenv(name)
+		return func() {
+			os.Setenv(name, env)
+		}
+	}
+
+	return func() {}
+}
+
+func TestExtractPrivateKeyAttributes(t *testing.T) {
+	if os.Getenv(t.Name()) != "1" {
```

Comment:
If I understand you correctly, you're saying instead of doing this:
```
if os.Getenv(t.Name()) != "1" {
```
I should do this:
```
if os.Getenv(t.Name()) != "" {
```
If that's the case, I agree. I'll update this.